lib/components/src/controls/options/Options.tsx

```diff
@@ -28,28 +28,36 @@ const normalizeOptions = (options: Options, labels?: Record<any, string>) => {
   return options;
 };
 
+const Controls: Record<string, FC> = {
+  check: CheckboxControl,
+  'inline-check': CheckboxControl,
```

Comment:
@shilman I think in the future we might want to rename these to `check-inline`, minor DX improvement maybe. Not 100% sure if it's worth it, but could be an addition so no breaking change, WDYT?